src/App.js

```diff
@@ -1,7 +1,12 @@
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
-  return <div className="App" />;
+  return (
+    <div className="App">
+      <Calculator />
+    </div>
+  );
 }
```

Comment:
- [ ] As part of the requirements, please make sure that your `App.js` component is a class-based component.

Reply:
@Lordkaito  I fixed it was very simple. I converted the function to class component-based